### DISCUSSION TITLE
Is possible to use treeshaking?

### DISCUSSION THREAD

**Discussion by aralroca** [dlemstra/magick-wasm#60] [Q&A]
I want to use this library in the browser only for one thing: convert a svg to eps format. 

I want to do this in the browser to don't create an API and consume less resources. However I see in bundlephobia that this library is 21mb, too much in my opinion for what I need, there are a lot of options of imagemagick that I never going to use.

In this case which is the best option? Is there any way to import only that feature of imagemagick?

Thank you so much and amazing library.

**Comment by dlemstra** [comment]
I don't think threeshaking will work. And there is not really a method that you can only import that specific part of the wasm library.
